### PR TITLE
Epochs decimation argument & package management

### PR DESCRIPTION
**Pull request by kambysese**
@Eric89GXL please review and comment. I ran into an error thrown by mne.Epochs becuase if was receiving an unacceptable argument `decim=0`; also while trying to setup mne python on a machine using system python I ran into the issue where the .externals package was not being installed apparently because it wasn't listed in `packages` for setup. The issue was fixed thanks to @ilogue by using setuptools.find_packages.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Thanks @kambysese

File: mnefun/_mnefun.py [hunk 1]
```diff
@@ -1356,7 +1356,7 @@ def do_preprocessing_combined(p, subjects):
             epochs = Epochs(raw, events, None, p.tmin, p.tmax,
                             baseline=_get_baseline(p), picks=picks,
                             reject=p.auto_bad_reject, flat=p.auto_bad_flat,
-                            proj=True, preload=True, decim=0)
+                            proj=True, preload=True, decim=1)
```
Comment (kambysese) [inline mnefun/_mnefun.py]: @Eric89GXL Did this somehow get rolled back? In current upstream/master this argument is reverted back to pre-PR.

File: setup.py [hunk 2]
```diff
@@ -35,7 +35,7 @@
           zip_safe=False,  # the package can run out of an .egg file
           classifiers=[],
           platforms='any',
-          packages=['mnefun', 'mnefun.tests'],
```
Comment (kambysese) [inline setup.py]: Same here.

**Comment by Eric89GXL** [comment]
Oh, you opened the PR into *your* master branch instead of `mnefun/master` :) Open a PR into the upstream branch and I'll merge it.

**Comment by kambysese** [comment]
Good lord! Not sure what is happening but I am not able to see or access my mnefun fork/branches when trying to create a new PR!

**Comment by Eric89GXL** [comment]
I think it's because I made `mnefun` public. You can delete your fork, re-fork, push this branch again, and you should be good. Just make sure you have local copies of any branches you want to make PRs with.

**Comment by kambysese** [comment]
lol jesus christ eric you're going to make me lose everything right now...

**Pull request merged**